README.md

```diff
@@ -781,6 +781,228 @@ client.delete_refund('321')
   ]
 }>
 ```
+
+### List customers
+
+#### Definition
+
+```ruby
+client.list_customers
+```
+
+#### Example Request
+
+```ruby
+require 'taxjar'
+client = Taxjar::Client.new(api_key: '48ceecccc8af930bd02597aec0f84a78')
+
+client.list_customers
+```
+
+#### Example Response
+
+```ruby
+['123', '124', '125']
```

Comment:
this currently returns the entire record for each customer

https://bold-resonance-9859.postman.co/collections/3636254-e7cac3be-254f-0869-fc3e-350c456fd7e1?workspace=4a985ccf-3cb9-4e97-868c-19b4687569d1#65ea8774-ef06-53b0-ad31-027d2447d3c7

Reply:
API endpoint response updated this week 👍